File: ABC/081/a_1.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
  // input
  int s;
  cin >> s;

  // compute
  int s1 = s / 100;
  int s2 = (s - s1*100) / 10;
  int s3 = s % 10;

  // output
  cout << s1 + s2 + s3 << endl;
}

File: abc/024/a_1.py
def main():
    # input
    A, B, C, K = map(int, input().split())
    S, T = map(int, input().split())

    # compute

    # output
    print(A*S+B*T if S+T<K else A*S+B*T-(S+T)*C)


if __name__ == '__main__':
    main()
